src/main/java/by/teachmeskills/page/RegistrationPage.java

```diff
@@ -0,0 +1,175 @@
+package by.teachmeskills.page;
+
+import by.teachmeskills.util.PropertiesLoader;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.Properties;
+
+import static by.teachmeskills.page.LoginPage.MAIN_PAIGE_LOGIN_BUTTON;
+
+public class RegistrationPage extends BasePage {
+    private static final By REGISTRATION_EMAIL = By.id("registration_email");
+    private static final By REGISTRATION_PASSWORD = By.id("registration_password");
+    private static final By REGISTRATION_PASSWORD_CONFIRMATION = By.id("registration_password_confirmation");
+    private static final By REGISTRATION_PASSWORD_HINT = By.id("registration_password_hint");
+    private static final By REQUIRED_CHECKBOX_1 = By.id("registration_terms_of_use");
+    private static final By REQUIRED_CHECKBOX_2 = By.id("registration_lost_password_warning_registered");
+    private static final By OK_BUTTON = By.xpath("//div[text()=' OK ']");
+    private static final By REGISTER_BUTTON = By.xpath("//a[text()=' Register ']");
```

Comment:
I've sent you an example of xpath //h1[contains(text(), 'Registration')]. Since text contains a lot of white spaces it is better to omit them  in locator by using 'contains(..)'